From f78ef4125d5ef94f4c3fd3ce3ff759ee551413e6 Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Mon, 9 Nov 2009 12:12:32 +0100
Subject: Template function overlap

---
 sfworks/common/IterTraits.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sfworks/common/IterTraits.h b/sfworks/common/IterTraits.h
index c273488..41c109a 100644
--- a/sfworks/common/IterTraits.h
+++ b/sfworks/common/IterTraits.h
@@ -97,7 +97,7 @@ template<> struct IterTagSelect<PtrIterTag> {
 };
 #endif
 
-template<> struct InpIterTraits<size_type> {
+template<> struct InpIterTraits<unsigned int> {
     typedef IntIterTag iterator_tag;
 };
 
-- 
1.6.6

