From bb640fa06b41128cef2682826971064d4318b928 Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Tue, 10 Nov 2009 10:24:42 +0100
Subject: Work around including two bison parsers in Tokenizer.cxx

---
 sfworks/xslt/impl/Tokenizer.cxx |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/sfworks/xslt/impl/Tokenizer.cxx b/sfworks/xslt/impl/Tokenizer.cxx
index eb91f52..fa34d6b 100644
--- a/sfworks/xslt/impl/Tokenizer.cxx
+++ b/sfworks/xslt/impl/Tokenizer.cxx
@@ -33,7 +33,10 @@
 #include "xslt/xsltParser.hpp"
 #ifdef USE_XPATH3
 # include "xpath/impl3/Tokenizer.h"
+#undef YYTOKENTYPE
+#define yytokentype xpath_tokentype
 # include "xpath/xpathParser.hpp"
+#undef yytokentype
 # include "xpath/impl3/ExprImpl.h"
 #else
 # include "xpath/impl/Tokenizer.h"
-- 
1.6.6

