From 5d3b53100cf8a9dcebf77e1d3fea0dee73a8ff9f Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Tue, 10 Nov 2009 15:02:07 +0100
Subject: Enable SIP_FEATURE_TYPE_STRUCT

---
 serna/pyapi/sip.t |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/serna/pyapi/sip.t b/serna/pyapi/sip.t
index 475adf4..f0feff9 100644
--- a/serna/pyapi/sip.t
+++ b/serna/pyapi/sip.t
@@ -7,9 +7,6 @@
     my $sip = get_package_info('sip', 'SIP');
     my @sip_ver = split '\.', `$sip -V`;
     my $sip_ver = (int(@sip_ver[0]) << 16) | (int(@sip_ver[1]) << 8) | int(@sip_ver[2]);
-    if ($sip_ver < 0x40800) {
-        Project("SIP_OPTIONS += -x TYPE_STRUCT");
-    }
     my $pyqt_sipflags = get_package_info('pyqt', 'PYQT_SIP_FLAGS');
     Project("SIP_OPTIONS += $pyqt_sipflags");
 #$}
-- 
1.6.6

