From 1f55384cc655a2cb1a41de58d4b2a3cd2f1516e1 Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Tue, 12 Jan 2010 11:29:24 +0100
Subject: Use (void *) when debug-outputting a pointer

---
 serna/docview/impl/EventTranslator.cxx |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/serna/docview/impl/EventTranslator.cxx b/serna/docview/impl/EventTranslator.cxx
index 75f2a94..d2bbd9f 100644
--- a/serna/docview/impl/EventTranslator.cxx
+++ b/serna/docview/impl/EventTranslator.cxx
@@ -55,7 +55,7 @@ void EventTranslator::registerEvent(const Sui::Action* action,
                                     EventMakerPtr eventMaker)
 {
     DBG(DV.EVENT) << "EventTranslator: register " << action->get(NAME)
-                  << " maker " << (int)eventMaker << std::endl;
+                  << " maker " << (void *)eventMaker << std::endl;
     (*eventMap_)[action] = eventMaker;
 }
 
@@ -138,7 +138,7 @@ void EventTranslator::registerActionExecutor(ActionExecutor* executor)
 {
     DBG(DV.EVENT) << "EventTranslator: register "  << executor << "/"
                   << executor->uiAction_->get(NAME)
-                  << " action-executor " << (int)executor << std::endl;
+                  << " action-executor " << (void *)executor << std::endl;
     (*actionExecutorMap_)[executor->uiAction()] = executor;
     executor->eventTranslator_ = this;
 }
-- 
1.6.6

