From d35efe6d0fc04c4cd1854a528edf92e48b6b1149 Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Sat, 17 Apr 2010 17:28:11 +0200
Subject: OnlineSpellerImpl::process_fo avoid segfault

---
 serna/plugins/speller/OnlineSpeller.cxx |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/serna/plugins/speller/OnlineSpeller.cxx b/serna/plugins/speller/OnlineSpeller.cxx
index 7eb2d17..d3c4410 100644
--- a/serna/plugins/speller/OnlineSpeller.cxx
+++ b/serna/plugins/speller/OnlineSpeller.cxx
@@ -188,6 +188,8 @@ static void update_fo_decoration(Formatter::TextFo* text_fo)
 
 bool OnlineSpellerImpl::process_fo()
 {
+    if (textFoModList_.isEmpty())
+        return false;
     Formatter::TextFo* text_fo = textFoModList_.pop_front();
     if (!text_fo) {
         deregisterHandler(this);
-- 
1.7.0.4

