From d73703afe46c746135fa18a7ac3165a64c3204c7 Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Sun, 10 Jan 2010 15:45:09 +0100
Subject: dash compliance: Do not use echo -e

---
 serna/app/script2.t |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/serna/app/script2.t b/serna/app/script2.t
index 6bde2c7..6945aab 100644
--- a/serna/app/script2.t
+++ b/serna/app/script2.t
@@ -80,7 +80,7 @@ $(VARSCRIPT): $(srcdir)/script2.t $(SH_ENV) $(TCSH_ENV) $(SERNA_WRAPPER)
 	chmod 755 $@
 	
 $(SERNA_WRAPPER):
-	@echo -e '\043!/bin/sh' > $@
+	@echo #$ $text = '\'#' . '!/bin/sh\' > $@';
 	@echo 'SDIR=`dirname $$0`' >> $@
 	@echo . '$${SDIR}/envserna.sh' >> $@
 	@echo exec '$${SDIR}/serna' '"$$@"' >> $@
-- 
1.6.6

