From 898031d5a135d33c61bedcfa2245219f1fab7ed4 Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Tue, 12 Jan 2010 11:42:44 +0100
Subject: Cast to (intptr_t) in Dav::DavManager->lock calls

---
 sfworks/grove/impl/Grove.cxx |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/sfworks/grove/impl/Grove.cxx b/sfworks/grove/impl/Grove.cxx
index 752474e..802617a 100644
--- a/sfworks/grove/impl/Grove.cxx
+++ b/sfworks/grove/impl/Grove.cxx
@@ -137,7 +137,7 @@ GROVE_EXPIMP void release_locks(Grove* g)
     if (g->groveBuilder()->flags() & 
         (GroveBuilder::doLocks|GroveBuilder::checkLocks)) 
             Dav::DavManager::instance().lock(Url("http://"),
-                Dav::DAV_UNLOCK, (int) g);
+                Dav::DAV_UNLOCK, (intptr_t) g);
 }
 
 Grove::~Grove()
@@ -307,10 +307,10 @@ static bool do_lock_resource(const String& url, const Grove* grove)
 {
     if (grove->groveBuilder()->flags() & GroveBuilder::doLocks)
         return Dav::DavManager::instance().lock(url, Dav::DAV_LOCK, 
-            (int) grove) != Dav::DAV_RESULT_OK;
+            (intptr_t) grove) != Dav::DAV_RESULT_OK;
     if (grove->groveBuilder()->flags() & GroveBuilder::checkLocks)
         return Dav::DavManager::instance().lock(url, Dav::DAV_CHECK_LOCK, 
-            (int) grove) == Dav::DAV_RESULT_LOCKED;
+            (intptr_t) grove) == Dav::DAV_RESULT_LOCKED;
     return false;
 }
     
-- 
1.6.6

