From ed001aa1a564a596edc377282761db0dce85b66e Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Tue, 12 Jan 2010 11:44:39 +0100
Subject: Cast to (long int) in sip stuff

---
 serna/pyapi/common/WrappedObject.sip |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/serna/pyapi/common/WrappedObject.sip b/serna/pyapi/common/WrappedObject.sip
index 4388e89..dee1dc8 100644
--- a/serna/pyapi/common/WrappedObject.sip
+++ b/serna/pyapi/common/WrappedObject.sip
@@ -16,7 +16,7 @@ public:
 %End
     int __int__();
 %MethodCode
-        sipRes = (int)sipCpp->getRep();
+        sipRes = (long int)sipCpp->getRep();
 %End
     bool operator==(const SimpleWrappedObject&) const;
     bool operator!=(const SimpleWrappedObject&) const;
-- 
1.6.6

